.changes/unreleased/tinymce-TINY-11313-2024-10-01.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Text and background color toolbar buttons would not be fully greyed out in readonly
```

Comment:
Just thinking out loud, should we refer to the buttons as: "Forecolor and backcolor toolbar buttons..."?

Reply:
we should use their config name, yes, good catch